OpenMiChroM/ChromDynamics.py

```diff
@@ -1299,8 +1299,32 @@ def createSimulation(self):
             print(energyInfo, file=f)
             print(f'\nPotential energy per forceGroup:\n {self.getForces()}', file=f)
 
+    def resetSimulation(self, keepforce = False):
```

Comment:
in the fuction  `resetSimulation   you declared, `keepforce = False` as argument but never use